libs/components/manifest/src/generator/generate-manifest.ts

```
@@ -61,12 +74,37 @@ export async function generateManifest(
     options.projectNames,
   );
 
-  const publicApi = await getPublicApi(projects);
-  const documentationConfig = await getDocumentationConfig(publicApi, projects);
+  const [publicApi, publicApiErrors] = await getPublicApi(projects);
+
+  const [documentationConfig, documentationConfigErrors] =
+    await getDocumentationConfig(publicApi, projects);
+
+  const [codeExamples, codeExamplesErrors] = await getCodeExamples(
+    publicApi,
+    documentationConfig,
+  );
+
+  const errors = [
+    ...publicApiErrors,
+    ...documentationConfigErrors,
+    ...codeExamplesErrors,
+  ];
+
+  if (errors.length > 0) {
+    throw new Error(
+      'The following errors were encountered when creating the manifest:\n - ' +
```

Since we're actually running TypeDoc for the unit tests, it's helpful to capture all possible errors with the manifest, instead of having to create a separate project for every possible error that can be encountered (if we simply threw the errors individually, all other errors would be ignored when compiling with TypeDoc).